jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/ResourceUtil.java

```diff
@@ -134,11 +134,18 @@ private static ObjectMapper getObjectMapper(ResourceFileType resourceFileType) {
                 .disable(SerializationFeature.WRITE_NULL_MAP_VALUES);
     }
 
-    public static File getFinalResourceDir(File resourceDir, String environment) {
-        if (resourceDir != null && StringUtils.isNotEmpty(environment)) {
-            return new File(resourceDir, environment);
+    public static List<File> getFinalResourceDirs(File resourceDir, String environmentAsCommaSeparateStr) {
+        List<File> resourceDirs = new ArrayList<>();
+
+        if (resourceDir != null && StringUtils.isNotEmpty(environmentAsCommaSeparateStr)) {
+            String[] environments = environmentAsCommaSeparateStr.split(",");
+            for (String environment : environments) {
+                resourceDirs.add(new File(resourceDir, environment.trim()));
+            }
+        } else if (StringUtils.isEmpty(environmentAsCommaSeparateStr)) {
```

Comment:
I know we were using `isNotEmpty` before, but maybe it's a better idea to change this to `isBlank` instead of `isEmpty`. This would prevent failures in case the environment is set to a blank string.